models/news.models.js

```diff
@@ -33,9 +50,11 @@ exports.selectArticles = () => {
     articles.created_at,
     articles.votes,
     articles.article_img_url,
+    ${bodyColumn}
```

Comment:
this may throw errors when there is no topic query as there would be a trailing comma on line 52, look at the notes for complex queries and how to build them up